source/sourcing_and_processing_data/processing.rst

```diff
@@ -1327,6 +1327,56 @@ It takes the following parameters:
     * Field
     * yes
 
+what3words
+^^^^^^^^^^
+
+Get 3 word address from coordinates. Requires a what3words API key to be configured by a domain administrator.
+
+It takes the following parameters:
+
+.. list-table::
+  :header-rows: 1
+
+  * * Label
+    * Description
+    * Type
+    * Mandatory
+  * * Language
+    * Field containing the language in which the 3 word address. Default is English
```

Comment:
the desired language for the 3 word address

Reply:
It's not a field, it's a string. Even if it's a FieldParam, it doesn't correspond to an actual field in the dataset.